prime-router/src/test/kotlin/fhirengine/translation/hl7/utils/CustomFHIRFunctionsTests.kt

```diff
@@ -562,7 +563,7 @@ class CustomFHIRFunctionsTests {
     }
 
     @Test
-    fun `test convertDateToAge in future`() {
+    fun `test convertDateToAge - add a day`() {
```

Comment:
Anything testing convertDateToAge specifically should probably get moved to a new test class specific to the converter class. This helps with organization and next time we have to extend/modify the worker class we don't have to touch code that does other stuff.